setup.py

```diff
@@ -318,14 +322,21 @@ def run(self):
 for data_dir in ["magic", "config", "plugins", "modules", "core"]:
     install_data_files.append("%s%s*" % (data_dir, os.path.sep))
 
+this_directory = os.path.abspath(os.path.dirname(__file__))
+with io.open(os.path.join(this_directory, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
 # Install the module, script, and support files
 setup(
     name=MODULE_NAME,
     version=MODULE_VERSION,
     description="Firmware analysis tool",
+    long_description=long_description,
+    long_description_content_type='text/markdown',
     author="Craig Heffner",
     url="https://github.com/ReFirmLabs/%s" % MODULE_NAME,
     requires=[],
+    python_requires=">=3",
```

Comment:
```suggestion
    python_requires=">=3.6",
```
* https://devguide.python.org/#status-of-python-branches
* https://devguide.python.org/devcycle/#end-of-life-branches

Reply:
Since I hadn't tested which version was the actual minimum required I just set it to `>=3`.

Reply:
If the Python core team has dropped support then this repo should too unless someone can express good reasons why the support risk is a worthwhile investment.

Reply:
Yeah, not really sure there, there tends to be a lot of legacy tooling floating around in the embedded world that could have implications.